src/style.css

```diff
@@ -0,0 +1,26 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-image: url(./images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  margin-right: 4px;
+}
+
+.stars--1 .stars__star:nth-child(1),
+.stars--2 .stars__star:nth-child(-n + 2),
+.stars--3 .stars__star:nth-child(-n + 3),
+.stars--4 .stars__star:nth-child(-n + 4),
+.stars--5 .stars__star:nth-child(n) {
```

Comment:
Here the modifier says that there will be 5 active stars. How many will be active if i decide to have 7 stars in each row?

Reply:
- [DEMO LINK](https://Uktam19980416.github.io/layout_stars/)
- [TEST REPORT LINK](https://Uktam19980416.github.io/layout_stars/report/html_report/)